source/computer/macos-setup.rst

```diff
@@ -160,6 +160,28 @@ C
 Command Line Tools for Xcode 已经提供了 C 编译器和相关工具，因而无需单独安装
 C 编译器。
 
+.. note::
+
+    Command Line Tools for Xcode 提供的 C 编译器本质上是 Apple Clang 编译器，
+    其与 GCC 编译器有差异，但足以满足日常科研中编译 C 程序的需求。因而一般
+    用户无需再安装 GCC 编译器。
```

Comment:
Better add official links for those two C compiles

Reply:
I can't find a better link for the apple clang.

Reply:
This link is good enough.